### PR TITLE
Last minute changes

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Api/PCS/Types/functionalRole.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export interface FunctionalRole {
+    Code: string;
+    Description: string;
+    Email: null;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/PCS/Types/functionalRole.ts:4]: string | null

File: src/apps/ScopeChangeRequest/Components/SearchableDropdown/PCSPersonRoleSearch.tsx [hunk 2]
```diff
@@ -18,22 +22,42 @@ export const PCSPersonRoleSearch = ({
 }: PCSLinkProps): JSX.Element => {
     const controller = useRef(new AbortController());
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchableDropdown/PCSPersonRoleSearch.tsx:23]: usecancellationTokwen

File: src/apps/ScopeChangeRequest/Components/Workflow/Criteria/Components/WorkflowCriteria.tsx [hunk 3]
```diff
@@ -131,15 +137,19 @@ export const WorkflowCriteria = ({
...
+            });
         };
         if (
             step.contributors &&
             step.contributors.some((x) => x.contribution === null) &&
             unsignedCriterias &&
             unsignedCriterias.length === 1
         ) {
-            spawnConfirmationDialog(
+            await spawnConfirmationDialog(
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Workflow/Criteria/Components/WorkflowCriteria.tsx:152]: remove await

File: src/apps/ScopeChangeRequest/Components/Workflow/Step/Components/WorkflowStep.tsx [hunk 4]
```diff
@@ -15,9 +15,9 @@ export function WorkflowStepContainer({ step }: WorkflowStepProps): JSX.Element
     const checkContributorAccess = () =>
         canAddContributor({ requestId: request.id, stepId: step.id });
     const { data: isAllowedToAddContributor } = useQuery(
-        `step/${step.id}`,
+        ['step', step.id],
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Workflow/Step/Components/WorkflowStep.tsx:18]: use querykey

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**